game/cards/dm06/colony_beetle.go

```diff
@@ -81,3 +81,64 @@ func FactoryShellQ(c *match.Card) {
 	})
 
 }
+
+func LivingCitadelVosh(c *match.Card) {
+
+	c.Name = "Living Citadel Vosh"
+	c.Power = 5000
+	c.Civ = civ.Nature
+	c.Family = []string{family.ColonyBeetle}
+	c.ManaCost = 5
+	c.ManaRequirement = []string{civ.Nature}
+	c.TapAbility = LivingCitadelVoshTapAbility
+
+	c.Use(fx.Creature, fx.Evolution, fx.TapAbility,
+		fx.When(fx.Summoned, func(card *match.Card, ctx *match.Context) {
+
+			// This is added for the case where LivingCitadelVosh is added to the field. There is another creature
+			// that doesn't initially have a tap abbility but should receive one. The change doesn't propagate fast
+			// enough to the FE and that creature doesn't get tap ability until another action takes places.
+			// This is an ugly workaround.
+			fx.FindFilter(
```

Comment:
This part is slightly ugly, unsure what's the better way to do this if there is one.